src/main/java/xyz/nucleoid/plasmid/mixin/game/rule/ServerPlayerEntityMixin.java

```diff
@@ -34,8 +34,8 @@ private void allowPvPInGames(CallbackInfoReturnable<Boolean> cir) {
         }
     }
 
-    @Inject(method = "stopRiding", at = @At("HEAD"), cancellable = true)
-    private void stopRiding(CallbackInfo ci) {
+    @Inject(method = "requestTeleportAndDismount", at = @At("HEAD"), cancellable = true)
+    private void requestTeleportAndDismount(CallbackInfo ci) {
```

Comment:
Needs testing, but the manual packet sending in here was to fix a desync that shouldn't be an issue anymore 🙂

Reply:
I think it would make sense to move this to Stimuli alongside an event for mounting an entity.

Reply:
Fixed in a96899577c2ec3f58085dd5cf3423d23ddc90ac8